src/algo/BubbleSort.js

```diff
@@ -221,17 +221,41 @@ export default class BubbleSort extends Algorithm {
 		this.compCount = 0;
 		this.swapCount = 0;
 		this.displayData = [];
+		this.listField.value = '';
```

Comment:
I don't have a strong preference as to whether or not the input field is cleared on "clear", but I noticed this is the only sorting algo that has this feature. 

Could you either remove this line or add it to the clearCallback for the other sorts? Just for consistency

Reply:
Thanks for finding this, I meant to make it clear input fields on all algos.

Related question, it is currently set up so that input fields are cleared when the algorithm is run, so users can't easily re-run the same inputs. I found this a bit annoying as a student. I think it makes more sense to _not_ clear inputs so that users can view/edit their inputs and re-run it, especially if the clear button does the job of resetting inputs. Do you have a preference for this?

Reply:
Will do :)

Reply:
Changes added!